java/test.sh

```diff
@@ -33,15 +33,18 @@ bazel build //java:gen_maven_deps
 echo "Build test jar."
 bazel build //java:all_tests_deploy.jar
 
+# Enable multi-worker feature in Java test
+TEST_ARGS=(-Dray.raylet.config.num_workers_per_process_java=10 -Dray.job.num-java-workers-per-process=10)
```

Comment:
Why need to set those parameters at the same time?

Reply:
Because we need to pass the tests both when multi-tenancy is enabled and disabled.